meshtastic/telemetry.proto

```diff
@@ -273,6 +273,26 @@ message LocalStats {
   uint32 num_total_nodes = 8;
 }
 
+/*
+ * Health telemetry metrics
+ */
+ message HealthMetrics {
+  /*
+   * Heart rate (beats per minute)
+   */
+  optional uint32 heart_bpm = 1;
+
+  /*
+   * SpO2 (blood oxygen saturation) level
+   */
+  optional uint32 spO2 = 2;
```

Comment:
We should probably have this comment list units, but I'm guessing percentage given the int_size of 8?

Reply:
That was my assumption